### PR TITLE
Uploaded In Assignment

### PR DESCRIPTION
**Pull request by NikitaShivhare1**

### CODE REVIEW

**Review by nileshp1122 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by nikita0699 — APPROVED** [review]

File: Nikita/Assignment_1/Assignment1.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<html>
+    <head>
+        <title>Assignment1</title>
+    </head>
+    <body>
+        <h1>Google
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment1.html]: remove extra line

File: Nikita/Assignment_1/Assignment1.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        <title>Assignment1</title>
+    </head>
+    <body>
+        <h1>Google
+            
+        </h1>
+        <h3>Introduction</h3>
+        <p>HTML is a Standard markup Language for Web Page</p>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment1.html]: extra line required at end of the file

File: Nikita/Assignment_1/Assignment2.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<HTML>
+    <Head>
+            <title>ASSIGNMENT2</title>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment2.html]: check indentation remove extra space only 4 spaces required

File: Nikita/Assignment_1/Assignment2.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    </Head>
+    <body>
+        <h1>HTML</h1>
+        <h3>Introduction</h3>
+        <p>HTML is Standard markup Language For Web Pages</p>
+        <h2>HTML Editor</h2>
+        <h5>Notepad Or TextEdit</h5>
+        <p>Web Pages can be Created and Modify By using Professional HTML Editors.<br>
+        Howerver, For Learning HTML we Recommanded A Simple TextEditor like Notepad(Pc) or TextEdit(MAC)
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment2.html]: check indentation

File: Nikita/Assignment_1/Assignment2.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        </p>
+    </body>
+</HTML>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment2.html]: extra line required at end of the file

File: Nikita/Assignment_1/Assignment3.html [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<HTML>
+    <Head>
+            <title>ASSIGNMENT3</title>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment3.html]: remove extra space

File: Nikita/Assignment_1/Assignment3.html [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+        <h3>Introduction</h3>
+        <p>HTML is Standard markup Language For Web Pages</p>
+        <hr>
+        <h2>HTML Editor</h2>
+        <h5>Notepad Or TextEdit</h5>
+        <p>Web Pages can be Created and Modify By using Professional HTML Editors.<br>
+        Howerver, For Learning HTML we Recommanded A Simple TextEditor like Notepad(Pc) or TextEdit(MAC)
+        </p>
+    </body>
+</HTML>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment3.html]: extra line required at end of the file

File: Nikita/Assignment_1/Assignment4.html [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<HTML>
+    <Head>
+            <title>ASSIGNMENT4</title>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment4.html]: check indentation

File: Nikita/Assignment_1/Assignment4.html [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+    </Head>
+    <body>
+        <h1>Nikita Shivhare</h1>
+        <h3>Introduction</h3>
+        <p>My Self Nikita Shivhare , I am From Burhanpur . I am Graduated From Davv University I have passed My 12th Exam in 2016 and I have passed my 10th in 2014 and I am Pursuing M.B.A with Davv University</p>
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment4.html]: remove extra line

File: Nikita/Assignment_1/Assignment4.html [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+    <Head>
+            <title>ASSIGNMENT4</title>
+    </Head>
+    <body>
+        <h1>Nikita Shivhare</h1>
+        <h3>Introduction</h3>
+        <p>My Self Nikita Shivhare , I am From Burhanpur . I am Graduated From Davv University I have passed My 12th Exam in 2016 and I have passed my 10th in 2014 and I am Pursuing M.B.A with Davv University</p>
+        
+    </body>
+</HTML>
```
Comment (KomalShinde813) [inline Nikita/Assignment_1/Assignment4.html]: extra line required at end of the file

File: Nikita/Assignment_2/Assignment_2.html [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+    <head>
+        <title>Assignment 02</title>
+    </head>
+    <body style="background-color: orange">
+        <center>
+            <h1>
+                Google
+            </h1>
+        </center>
+                <pre><em>This article is about the company.</em></pre>
```
Comment (KomalShinde813) [inline Nikita/Assignment_2/Assignment_2.html]: check indentation

File: Nikita/Assignment_2/Assignment_2.html [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+                </p>
+                <p>
+                    At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins>
+                    years, until the year 2024.
+                </p>
+                <center>
+                    <bold style="font-size: 32px;color: green">Thank You !!!</bold>
+                </center>
+    </body>
+</html>
```
Comment (KomalShinde813) [inline Nikita/Assignment_2/Assignment_2.html]: extra line required at end of the file

File: Nikita/Assignment_2/Assignment_2.html [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+                <h2>Initial public offering</h2>
+                <p>
+                    Google's initial public offering (IPO) took place five years later, on 19th August 2004.
+                </p>
+                <p>
+                    At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for <del>30</del> <ins>20</ins>
+                    years, until the year 2024.
+                </p>
+                <center>
+                    <bold style="font-size: 32px;color: green">Thank You !!!</bold>
```
Comment (KomalShinde813) [inline Nikita/Assignment_2/Assignment_2.html]: space required after each style property

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 14]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:7]: change title

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 15]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:3]: check indentation

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+    <h1>Ambersoft</h1>
+    <ol>
+        <li>MI-MEAN</li>
+                <ol style="list-style: lower-roman;">
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:13]: check indentation only 4 space required from parent tag

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+
+                        <li>MI-MEAN-9
+                            <ul>
+                                <li style="list-style-type:circle;">Anil</li>
+                                <li style="list-style-type:circle;">Manoj</li>
+                            </ul>
+                        </li>
+                    </li>
+                </ol>
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:45]: remove extra line

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 18]
```diff
@@ -0,0 +1,67 @@
...
+        
+        <li>SYSTEM</li>
+               <ol style="list-style: lower-roman; ">
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:48]: check indentation only 4 space required from parent tag

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+                <li>SYSTEM-1</li>
+            <ul>
+                <li style="list-style-type:circle;">Anu</li>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:51]: space required after : in style

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+                <li style="list-style-type:circle;">Dipti</li>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:52]: space required after : in style

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 21]
```diff
@@ -0,0 +1,67 @@
...
+            </ul>
+
+            <li>SYSTEM-2</li>
+            <ul>
+                <li style="list-style-type:circle;">Triveni</li>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:57]: space required after : in style

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 22]
```diff
@@ -0,0 +1,67 @@
...
+                <li style="list-style-type:circle;">Kalyani</li>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:58]: space required after : in style

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 23]
```diff
@@ -0,0 +1,67 @@
...
+            </ul>
+
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:61]: remove extra lines

File: Nikita/Assignment_3/NikitaA3_1.html [hunk 24]
```diff
@@ -0,0 +1,67 @@
...
+
+            </ol>
+    </ol>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_1.html:66]: extra line required at the end of file

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 25]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: check indentation

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 26]
```diff
@@ -0,0 +1,50 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment 3 --Question 2</title>
+</head>
+<style>
+    body{
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: space required before {

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 27]
```diff
@@ -0,0 +1,50 @@
...
+        text-align: center;
+        background-color: cadetblue;
+    }
+    a:link{
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: space required before {

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 28]
```diff
@@ -0,0 +1,50 @@
...
+        color:cyan;
+    }
+    a:visited{
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: space required before {

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 29]
```diff
@@ -0,0 +1,50 @@
...
+        color: greenyellow;
+    }
+</style>
+<body>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: check indentation

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 30]
```diff
@@ -0,0 +1,50 @@
...
+    <div style="text-align: center;">
+        <h1>AmberSoft</h1><br>
+        <a href="#" >Home </a> |
+        <a href="MI-MEAN.html" >MI-MEAN </a> |
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: remove extra space

File: Nikita/Assignment_3/NikitaA3_2.html [hunk 31]
```diff
@@ -0,0 +1,50 @@
...
+        <div style="width: 33.33% height: auto;;">
+            <img src="Images/Panda.jpg" alt="image" height="300px" width="400px" style="margin: 10px;">
+        </div>
+    </div>
+    <div>
+        <h2 style="text-align: left;">Mean Stack Development</h2>
+        <p>This program will advance your career as a MEAN stack developer. You’ll learn top skills such as MongoDB, Express.js, Angular, and Node.js (“MEAN”), plus GIT, HTML, CSS, and JavaScript to build and deploy interactive applications and services.</p>
+    </div>
+</body>
+</html>
```
Comment (KomalShinde813) [inline Nikita/Assignment_3/NikitaA3_2.html]: extra line required at the end of file

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 32]
```diff
@@ -0,0 +1,57 @@
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: remove extra line

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 33]
```diff
@@ -0,0 +1,57 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Question 1</title>
+    <style>
+        caption{
+            font-size: x-large;
+            font-style: italic;
+            background-color:rosybrown;
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: space required

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 34]
```diff
@@ -0,0 +1,57 @@
...
+        }
+        table, th, td {
+            border: 1px solid black;
+            border-collapse: collapse;
+            background-color:rosybrown;
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: space required

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 35]
```diff
@@ -0,0 +1,57 @@
...
+            padding :14px;
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: remove space before : and add after :

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 36]
```diff
@@ -0,0 +1,57 @@
...
+        }
+
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: remove extra lines

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 37]
```diff
@@ -0,0 +1,57 @@
...
+
+    </style>
+</head>
+<body style="margin: 20px 200px;">
+        <table style="width: 70%;">
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: check indentation only 4 spaces allowed from parent tag

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 38]
```diff
@@ -0,0 +1,57 @@
...
+
+            <div>
+            <caption>A test table with merged cells</caption>
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html]: check indentation

File: Nikita/Assignment_4/NikitaA4_1.html [hunk 39]
```diff
@@ -0,0 +1,57 @@
...
+                <td>0.003</td>
+                <td>40%</td>
+            </tr>
+            <tr>
+                <th>Females</th>
+                <td>1.7</td>
+                <td>0.002</td>
+                <td>43%</td>
+            </tr>  
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_1.html:31]: remove extra line

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 40]
```diff
@@ -0,0 +1,75 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: check indentation

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 41]
```diff
@@ -0,0 +1,75 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment2</title>
+    <style>
+        table, th, td{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 42]
```diff
@@ -0,0 +1,75 @@
...
+            border-left: 12px solid grey;
+            border-right: 12px solid grey;
+            border-top: 2px solid grey;
+            border-bottom: 2px solid grey;
+            padding: 15px;
+            border-collapse: collapse;
+        }
+
+        .a{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 43]
```diff
@@ -0,0 +1,75 @@
...
+            border-top: 12px solid grey;
+        }
+
+        .b{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 44]
```diff
@@ -0,0 +1,75 @@
...
+            border-top: transparent;
+        }
+
+        .c{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 45]
```diff
@@ -0,0 +1,75 @@
...
+            border-bottom: 12px solid grey;
+        }
+
+    </style>
+</head>
+<body style="margin: 50px 50px;">
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: check indentation

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 46]
```diff
@@ -0,0 +1,75 @@
...
+    <table style="width: 50%;">
+
+        <caption style="color:mediumpurple;"><h1>Jan's Spring Term Afternoon Schedule</h1></caption>
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: space required in style

File: Nikita/Assignment_4/NikitaA4_2.html [hunk 47]
```diff
@@ -0,0 +1,75 @@
...
+            <td>  </td>
+            <td class="b">  </td>
+            <td class="b">  </td>
+        </tr>
+        <tr>
+            <th>3.00</th>
+            <td rowspan="3">EC<br>Meeting</td>
+            <td> </td>
+            <td> </td>
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_2.html]: where is closing tags

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 48]
```diff
@@ -0,0 +1,87 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: check indentation

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 49]
```diff
@@ -0,0 +1,87 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html:7]: change title

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 50]
```diff
@@ -0,0 +1,87 @@
...
+    <style>
+        table{
+            border: 1px solid black;
+
+        }
+        th, td{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 51]
```diff
@@ -0,0 +1,87 @@
...
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            border-right: 1px solid gainsboro;
+            padding: 5px;
+            border-width:2px;
+        }
+        .a{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 52]
```diff
@@ -0,0 +1,87 @@
...
+            border-top: 1px solid black;
+            border-left: 1px solid black;
+            border-bottom: 1px solid gainsboro;
+            padding: 15px;
+        }
+        .b{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 53]
```diff
@@ -0,0 +1,87 @@
...
+            background-color:cornsilk;
+        }
+        .c{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 54]
```diff
@@ -0,0 +1,87 @@
...
+            background-color:lightgray
+        }
+        .d{
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: space required before {

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 55]
```diff
@@ -0,0 +1,87 @@
...
+        <tr>
+            <td rowspan="3">Tuesday</td>
+            <td class="b">8:00am</td>
+            <td class="b">11:00am</td>
+            <td rowspan="3"><div><div class="a">XPath</div><div class="a">XSL Transformations</div></div></td>
+        </tr>
+        <tr>
+            <td class="b">11:00am</td>
+            <td class="d">2:00pm</td>
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: remove extra line

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 56]
```diff
@@ -0,0 +1,87 @@
...
+        </tr>
+        <tr>
+            <td class="d">2:00pm</td>
+            <td class="c">5:00pm</td>
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: remove extra line

File: Nikita/Assignment_4/NikitaA4_3.html [hunk 57]
```diff
@@ -0,0 +1,87 @@
...
+        </tr>
+        <tr>
+            <td>Wednesday</td>
+            <td class="b">8:00am</td>
+            <td class="d">12:00pm</td>
+            <td>XSL Formatting Objects</td>
+        </tr>
+    </table>
+
```
Comment (KomalShinde813) [inline Nikita/Assignment_4/NikitaA4_3.html]: remove extra line

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Pull request merged**